### PR TITLE
Lodz page implementation

### PR DESCRIPTION
**Pull request by iAreStilos**

### CODE REVIEW

File: public/city/lodz.jpg
Comment (ssynowiec) [inline public/city/lodz.jpg:1]: Do we have permission to use this graphic?
Arc 2024-09-03 11 22 39

File: src/app/(cities)/lodz/page.mdx [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
 };
 
-# Łódź
\ No newline at end of file
+<CityBanner city="Łódź" background="city/lodz.jpg" />
+
+<LocalGroups
+    localGroups={[
+        'https://www.meetup.com/meet-js-lodz/',
+        'https://lu.ma/user/usr-eNzoaGXYjizOBQI',
```
Comment (ssynowiec) [inline src/app/(cities)/lodz/page.mdx:52]: We can now add a link to lu.ma with a city filter : https://lu.ma/meet.js?tag=%C5%82%C3%B3d%C5%BA but we don't have to (if you prefer)

Comment (ssynowiec) [inline src/app/(cities)/lodz/page.mdx:52]: We can now add a link to lu.ma with a city filter https://lu.ma/meet.js?tag=%C5%82%C3%B3d%C5%BA but we don't have to (if you prefer)

File: public/city/lodz.jpg
Comment (iAreStilos) [reply public/city/lodz.jpg:1]: image

File: src/app/(cities)/lodz/page.mdx (hunk 1 above)
Comment (iAreStilos) [reply src/app/(cities)/lodz/page.mdx:52]: I think it is fine as it is now :)

**Pull request merged**